components/console_cmd_ping/idf_component.yml

```diff
@@ -0,0 +1,8 @@
+version: 1.0.0
+url: https://github.com/espressif/esp-protocols/tree/master/components/console_cmd_ping
+description: The component provides a console where the 'ping' command can be executed.
+dependencies:
+  idf:
+    version: '>=5.0'
+  espressif/console_simple_init:
```

Comment:
console_simple_init is not used by this component. It is added so that a user can access `console_cmd_init()` and `console_cmd_start()` without having to explicitly include any other component.

Reply:
console_cmd_ping now uses console_simple_init component for automatic registration of console commands.